prime-router/build.gradle.kts

```diff
@@ -810,4 +810,4 @@ dependencies {
     testImplementation("com.willowtreeapps.assertk:assertk-jvm:0.25")
     testImplementation("io.ktor:ktor-client-mock:$ktorVersion")
     testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.9.1")
-}
+}
```

Comment:
This probably should not be part of the PR (although testing the hooks might have done this).